plugins/chat/app/models/chat/thread.rb

```diff
@@ -72,7 +79,10 @@ def self.update_counts
         ) subquery
         WHERE threads.id = subquery.thread_id
         AND subquery.replies_count != threads.replies_count
+        RETURNING threads.id AS thread_id;
       SQL
+      return if updated_thread_ids.empty?
```

Comment:
Don't need to bother trying to clear the cache if all the DB values are up to date